ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -136,7 +182,89 @@ const EditGasItem = ({ priorityLevel, onClose }) => {
         )}
       </span>
       <span className="edit-gas-item__tooltip">
-        <InfoTooltip position="top" />
+        <InfoTooltip
+          contentText={
+            <div className="edit-gas-item__tooltip__container">
+              {priorityLevel !== PRIORITY_LEVELS.CUSTOM &&
+              priorityLevel !== PRIORITY_LEVELS.DAPP_SUGGESTED ? (
+                <img
+                  alt=""
+                  width={130}
+                  src={`./images/curve-${priorityLevel}.svg`}
+                />
+              ) : null}
+              {priorityLevel === PRIORITY_LEVELS.HIGH ? (
+                <div className="edit-gas-item__tooltip__container__dialog">
+                  <Typography fontSize="12px" color={COLORS.WHITE}>
```

Comment:
`0.75rem` instead?

Reply:
unfortunately `rems` don't appear to work as a prop to the `Typography` component...